.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,84 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+labels: ["bug", "triage"]
+projects: ["freedomofpress/dangerzone"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi, and thanks for taking the time to open this bug report. 
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Can you specify the steps you followed, so we can reproduce?
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: dropdown
+    id: os-name
+    attributes:
+      label: OS Name
+      description: Which Operating System are you using?
+      options:
+        - Windows
+        - MacOS
+        - Linux
+    validations:
+      required: true
+  - type: textarea
+    id: os-version
+    attributes:
+      label: Operating System Version
+      description: |
+        Which version of your operating sytem do you use?
+
+        - Follow these links to find out the OS version (on [Windows](https://learn.microsoft.com/en-us/windows/client-management/client-tools/windows-version-search), on [OSX](https://support.apple.com/en-us/109033))
+        - On Linux, you should have good results with this command `cat /etc/os-release`
+    validations:
+      required: true
+  - type: textarea
+    id: dangerzone-version
+    attributes:
+      label: Dangerzone version
+      description: Which version of Dangerzone are you using?
+    validations:
+      required: true
+  - type: textarea
+    id: docker-desktop-version
+    attributes:
+      label: Docker Desktop version
+      description: |
+        On Windows and MacOS, if you're using Docker Desktop, which version is it?
+        Please give us the full output of this command: `docker version`
+
+        This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: textarea
+    id: podman-version
+    attributes:
+      label: Podman version
+      description: |
+        On Linux, which version of podman are you using?
+        Please give us the full output of this command: `podman version`
+
+        This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: textarea
+    id: logs
+    attributes:
+      label: Additional log output
```

Comment:
From what I've seen, the additional info field in GitHub issues is usually free-form, so that the user can provide some extra context. Perhaps we can keep it that way here as well, and ask for command output explicitly, in a separate field.

For instance:

---

**Podman/Docker info**

If the bug occurs during document conversion, or is otherwise related with Docker/Podman, please copy and paste the following commands in your terminal, and provide us with the output:

```shell
docker version
docker info -f 'json'
docker images
docker run hello-world
```

_replace `docker` with `podman` where necessary_

---

**Document conversion logs**

_For Windows:_

If the bug occurs during document conversion, we'd like some logs from this process. These logs will require some basic familiarity with the Windows terminal.

First, copy the path of your document, with right click -> Copy as path. Then, please run the following commands, and provide us with the output.

Perform the conversion via the Dangerzone CLI (replace `\path\to\file` with the path you copied)

```
& 'C:\Program Files (x86)\Dangerzone\dangerzone-cli.exe' \path\to\file
```

Start the conversion process with debug logs:

```
& 'C:\Program Files\Docker\Docker\resources\bin\docker.EXE' run -e RUNSC_DEBUG=1 --security-opt=no-new-privileges:true --cap-drop all --cap-add SYS_CHROOT --network=none -u dangerzone --rm -i dangerzone.rocks/dangerzone /usr/bin/python3 -m dangerzone.conversion.doc_to_pixels
```

_For MacOS:_

If the bug occurs during document conversion, we'd like some logs from this process. Please copy and paste the following commands in your terminal, and provide us with the output:


Perform the conversion via the Dangerzone CLI (replace `/path/to/file` with the path to your document):

```
/Applications/Dangerzone.app/Contents/MacOS/dangerzone-cli /path/to/file
```

Start the conversion process with debug logs:

```
docker run --security-opt=no-new-privileges:true --cap-drop all --cap-add SYS_CHROOT --network=none -u dangerzone -e SC_DEBUG=1 --rm -i dangerzone.rocks/dangerzone /usr/bin/python3 -m dangerzone.conversion.doc_to_pixels
```

_For Linux:_

If the bug occurs during document conversion, we'd like some logs from this process. Please copy and paste the following commands in your terminal, and provide us with the output:


Perform the conversion via the Dangerzone CLI (replace `/path/to/file` with the path to your document):

```
dangerzone-cli /path/to/file
```

Start the conversion process with debug logs:

```
podman run --security-opt=no-new-privileges:true --cap-drop all --cap-add SYS_CHROOT --network=none -u dangerzone -e SC_DEBUG=1 --rm -i dangerzone.rocks/dangerzone /usr/bin/python3 -m dangerzone.conversion.doc_to_pixels
```

_(the above commands need further tweaking, to work with the recent seccomp changes)_

---

**Additional info**

Please provide us with any additional info, such as logs, extra context, that may help us debug this issue

Reply:
Udpated in 498dcc9152c050e23c23fab562a8db1621e8c90a